src/Oscoin/Crypto.hs

```diff
@@ -0,0 +1,8 @@
+module Oscoin.Crypto
+    ( RealWorldCrypto
+    , MockCrypto(..)
+    ) where
+
+data RealWorldCrypto
```

Comment:
I think it makes sense to call this just `Crypto`, since "real-world-ness" is implied in the rest of the codebase - but we can do this in a later PR.

Reply:
A comment explaining what it's for would also be nice.

Reply:
Not particularly attached to the `RealWorldCrypto`, so I am happy to rename it if `Crypto` is more fitting 😉

Reply:
I'd be in favor of just `Crypto`, because we don't have stuff like `RealWorldGossip` and `RealWorldConsensus` in the codebase, so might raise eyebrows.

Reply:
cf21232